buyer-api/test/operations/saveSeller.test.js

```diff
@@ -9,7 +9,12 @@ it('saves seller id', async (assert) => {
   assert.true(sellers.put.called);
 });
 
-it('doesn`t save id when address already registered', async (assert) => {
+it('returns true when id is same as saved', async (assert) => {
+  await saveSeller('2', 3);
```

Comment:
I'd also add the assertion:
```suggestion
  const response = await saveSeller('2', 3);
  assert.true(response);
```